src/scene/light.js

```diff
@@ -152,6 +153,8 @@ class Light {
         this._innerConeAngleCos = Math.cos(this._innerConeAngle * Math.PI / 180);
         this._outerConeAngleCos = Math.cos(this._outerConeAngle * Math.PI / 180);
 
+        this._usesPhysicalUnits = false;
```

Comment:
I'd set this to undefined, so that it's not false nor true .. otherwise the value needs to be in sync with the initial setting in the scene.